internal/cli/apps.go

```diff
@@ -225,6 +225,7 @@ func listAppsCmd(cli *cli) *cobra.Command {
 				cmd.Context(),
 				inputs.Number,
 				func(opts ...management.RequestOption) (result []interface{}, hasNext bool, err error) {
+					opts = append(opts, management.Parameter("is_global", "false")) // Excluding deprecated global client from list
```

Comment:
```suggestion
					opts = append(opts, management.Parameter("is_global", "false"))
```
Removing the comment as the global client isn't deprecated.